app/src/db/models/submission.ts

```diff
@@ -26,6 +26,7 @@ export default {
       project_description: input.projectDescription,
       submitted_at: new Date(input.submittedAt ?? Date.now()),
       submitted_by: input.submittedBy,
+      consent_to_feedback: input.feedbackConsent,
```

Comment:
Let's try to keep variable naming as consistent as possible between the front end back ends when we can.
`feedbackConsent` -> `consentToFeedback`